src/gdb.ts

```diff
@@ -97,7 +98,7 @@ class GDBDebugSession extends MI2DebugSession {
 				this.sendErrorResponse(response, 102, `Failed to SSH: ${err.toString()}`);
 			});
 		} else {
-			this.miDebugger.load(args.cwd, args.target, args.arguments, args.terminal).then(() => {
+			  this.miDebugger.load(untildify(args.cwd), args.target, args.arguments, args.terminal).then(() => {
```

Comment:
should this really be done in gdb/lldb.ts instead of in the mi engine? You also changed the indentation on most lines by accident.